src/context/WalletProvider/NativeWallet/types.ts

```diff
@@ -19,3 +20,7 @@ export interface NativeSetupProps
   vault: Vault
   dispatch: React.Dispatch<ActionTypes>
 }
+
+export interface LoginError {
+  message: string
```

Comment:
can we tighten this up to a string union if we're using logic on it above?

Reply:
:heavy_check_mark: